src/include/host/topology.h

```diff
@@ -63,7 +63,6 @@
 #define SOF_TKN_COMP_PERIOD_SINK_COUNT          400
 #define SOF_TKN_COMP_PERIOD_SOURCE_COUNT        401
 #define SOF_TKN_COMP_FORMAT                     402
-#define SOF_TKN_COMP_PRELOAD_COUNT              403
```

Comment:
We should not remove a token and reassign it to something else later. If we decide not to use it, them mark is as no longer used so that the next token is 404.

Reply:
got it. will add comment that this token is not used any more.